### PR TITLE
CICD for otterscan staging

### PR DESCRIPTION
**Pull request by rrw-zilliqa**

### CODE REVIEW

File: cd/base/frontend-config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: developer-portal
+  namespace: developer-portal
```
Comment (frankmeds) [inline cd/base/frontend-config.yaml:5]: unwanted value probable from a copy and paste

File: cd/base/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: developer-portal
```
Comment (frankmeds) [inline cd/base/namespace.yaml:4]: perhaps from a copy and paste

File: cd/overlays/staging/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          kubernetes.io/ingress.class: gce
+          kubernetes.io/ingress.global-static-ip-name: otterscan-zilstg-dev
+          networking.gke.io/managed-certificates: otterscan
+  - target:
+      kind: Deployment
+      name: otterscan
+    patch: |-
+      - op: replace
+        path: "/spec/template/spec/containers/0/env/0/value"
+        value: "https://api.zq2-prototestnet.zilliqa.com"
```
Comment (frankmeds) [inline cd/overlays/staging/kustomization.yaml:28]: is the zq2 prototestnet api the wanted value in staging?

File: cd/base/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: "otterscan"
+    spec:
+      containers:
+        - image: otterscan
+          name: otterscan
+          env:
```
[the remark this answers is not in the document]
Comment (rrw-zilliqa) [reply cd/base/deployment.yaml:23]: Nope - I can add it from the overlay ..

File: cd/base/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - namespace.yaml
+  - deployment.yaml
+  - svc.yaml
+  - ingress.yaml
+  - backend-config.yaml
```
[the remark this answers is not in the document]
Comment (rrw-zilliqa) [reply cd/base/kustomization.yaml:8]: I copied it from dev-portal; can you point me to where I should've copied it from?

File: cd/overlays/staging/kustomization.yaml (hunk 3 above)
Comment (rrw-zilliqa) [reply cd/overlays/staging/kustomization.yaml:28]: Well, we have to point it at something - this was the least worst something I had ..

Comment (frankmeds) [reply cd/overlays/staging/kustomization.yaml:28]: my question is more related to understand if this it the prototestnet instance or just a test instance

**Comment by rrw-zilliqa** [comment]
We decided to do this via LocalStorage, so closing this PR.

**Pull request closed without merging**